Cache/Hashing.lean

```diff
@@ -60,14 +61,25 @@ def HashMemo.filterByFilePaths (hashMemo : HashMemo) (filePaths : List FilePath)
 /-- We cache the hash of each file and their dependencies for later lookup -/
 abbrev HashM := StateT HashMemo CacheM
 
-/-- Gets the file paths to Mathlib files imported on a Lean source -/
-def getFileImports (source : String) (pkgDirs : PackageDirs) : Array FilePath :=
-  let s := Lean.ParseImports.main source (Lean.ParseImports.whitespace source {})
-  let imps := s.imports.map (·.module.components |> .map toString)
-    |>.filter fun parts => match parts.head? with
-      | some head => pkgDirs.contains head
-      | none => false
-  imps.map (mkFilePath · |>.withExtension "lean")
+/--
+Read the imports from the raw file `content` and return an array of tuples
+`(module name, source file)`.
+Note: `mod` is the name of the parsed module and is only used for displaying an error
```

Comment:
Right now, you refer to the first parameter twice, in different ways. Perhaps speak of `module_name` twice?

Reply:
I've updated the docstring